### PR TITLE
Exports a MT to GCP bucket

### PR DESCRIPTION
**Pull request by hopedisastro**

### CODE REVIEW

File: str/helper/mt_extractor.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    '--file-path',
+    help='GCS file path to BGZIP VCF file.',
+    type=str,
+)
+@click.command()
+def main(file_path):
+    """writes a BGZIP VCF as a Hail Matrix Table to a GCS bucket"""
+
+    init_batch()
+    gcs_path = output_path('str.mt', 'analysis')
```
Comment (MattWellie) [inline str/helper/mt_extractor.py:29]: I'm not super wild about this, does all this data need to be in an analysis bucket? I'm mostly concerned that this script becomes a way of taking any VCF data and transcribing it to a less protected bucket

File: str/helper/mt_extractor.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+# pylint: disable=missing-function-docstring,no-member
```
Comment (MattWellie) [inline str/helper/mt_extractor.py:2]: I'm not sure either of these are relevant, you have one function with a docstring

File: str/helper/mt_extractor.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import hail as hl
+import click
+
+from cpg_utils.hail_batch import output_path, init_batch
+
+
+@click.option(
+    '--file-path',
+    help='GCS file path to BGZIP VCF file.',
+    type=str,
```
Comment (MattWellie) [inline str/helper/mt_extractor.py:22]: Not important, but the default type of a click arg is str, so this shouldn't need clarification

**Review by MattWellie — CHANGES_REQUESTED** [review]

File: str/helper/mt_extractor.py (hunk 2 above)
Comment (hopedisastro) [inline str/helper/mt_extractor.py:2]:
```suggestion
```

File: str/helper/mt_extractor.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
```
Comment (hopedisastro) [inline str/helper/mt_extractor.py:21]:
```suggestion
```

File: str/helper/mt_extractor.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+@click.option(
+    '--file-path',
+    help='GCS file path to BGZIP VCF file.',
+)
+@click.command()
+def main(file_path):
+    """writes a BGZIP VCF as a Hail Matrix Table to a GCS bucket"""
+
+    init_batch()
+    gcs_path = output_path('str.mt', 'analysis')
```
Comment (hopedisastro) [inline str/helper/mt_extractor.py:27]:
```suggestion
    gcs_path = output_path('str.mt')
```

File: str/helper/mt_extractor.py (hunk 1 above)
Comment (hopedisastro) [reply str/helper/mt_extractor.py]: removed analysis from output bucket

File: str/helper/mt_extractor.py (hunk 2 above)
Comment (hopedisastro) [reply str/helper/mt_extractor.py]: removed

File: str/helper/mt_extractor.py (hunk 3 above)
Comment (hopedisastro) [reply str/helper/mt_extractor.py]: removed

**Review by MattWellie — APPROVED** [review]
LGTM!

**Pull request merged**